### DISCUSSION TITLE
Setting home zone radius

### DISCUSSION THREAD

**Discussion by SiliconAvatar** [home-assistant/frontend#11170] [Other feature requests]
**Home Assistant release with the issue:**

0.94.2
**Last working Home Assistant release (if known):**
N/A
**UI (States or Lovelace UI?):**

Lovelace
**Browser and Operating System:**

Chrome 74.0.3729.169
**Description of problem:**

Using the new configuration page to set the home zone, it does not allow setting a Radius for the home zone. I realize this can be overridden in the `configuration.yaml` file, however for this UI config to fully supplant the YAML configuration, the ability to set the radius and icon should be available from the GUI as well.

**Javascript errors shown in the web inspector (if applicable):**
```

```

**Additional information:**

**Comment by steverep** [comment]
This is related to #5991 because, in general, you cannot edit the home zone the same way as other zones which makes it much less accessible.
